### ISSUE TITLE
No IMU intrinsics published by T265 node

### ISSUE THREAD

**Issue by schmidtp1** [realsenseai/realsense-ros#739]
The node advertises `/camera/accel/imu_info` and `/camera/gyro/imu_info`
but no messages arrive,
for example:
```
$ rostopic hz /camera/accel/imu_info
subscribed to [/camera/accel/imu_info]
no new messages
no new messages
no new messages
```

**Comment by doronhi** [comment]
Thanks. I add #809 but before merging, I have a doubt:
All the imu_info messages are the same so is it advisable to publish them as a topic or should we have a service for that? 
Any thoughts on the subject?

**Comment by doronhi** [comment]
Add #810 - solution with services.

**Comment by schmidtp1** [comment]
I would prefer them to be published to make sure to have them in the bag when recording.
Maybe they could be "latched"?

**Comment by doronhi** [comment]
I like the idea. I fixed PR #809. What do you think?
